index.js

```diff
@@ -70,33 +71,51 @@ function getData() {
             var lonRow = parseFloat(line.split(',')[lon]),
                 latRow = parseFloat(line.split(',')[lat]);
             if (lonRow > -180 || lonRow < 180 || latRow > -85 || latRow < 85 || (lonRow !== 0 && latRow !== 0))
-                collection.push(turf.point(lonRow, latRow));
-            else console.log("Invalid GEOM skipped");
+                collection.push(turf.point([lonRow, latRow]));
+            else console.error("Invalid GEOM skipped");
         }
     });
-
     rl.on('close', getOSM);
 }
 
 function getOSM() {
     fc = turf.featurecollection(collection);
     envelope = turf.envelope(fc);
-    tiles = cover.geojson(envelope.geometry, { min_zoom: 7, max_zoom: 7 });
-    var queries = [];
+    console.error('Creating tiles');
+    var tiles = cover.geojson(envelope.geometry, { min_zoom: 7, max_zoom: 7 });
+    var urls = []
+
     async.each(tiles.features, function(feat, cb) {
         var bbox = turf.extent(feat);
-        var query = overpass + encodeURIComponent("[out:json][timeout:25];(node[" + tag.key +  "=" + tag.value + "]("+bbox[1]+","+bbox[0]+","+bbox[3]+","+bbox[2]+"););out body;>;out skel qt;");
-        request(query, function(err, res, body) {
-            if (err || res.statusCode !== 200) cb(new Error("Overpass Query Failed!\n" + body));
-            JSON.parse(body).elements.forEach(function(osmfeat) {
-                osmcollection.push(turf.point(osmfeat.lon, osmfeat.lat));
-            });
-            setTimeout(cb, 5000); //Respect Overpass limits
```

Comment:
I don't think we need this because it only issues the next request after it receives the current one. If we still do, where should I put it?